docs/resources/topic.md

```diff
@@ -0,0 +1,30 @@
+# gitlab\_topic
```

Comment:
Please use the `Description` fields in the resource schema definition and run `make reviewable` locally - this will generate that markdown file for you!